pysprint_tracks.py

```diff
@@ -145,22 +146,25 @@ def __init__(self):
         self.wrench_position = None
         self.on_bridge_or_ramp_wrench = False
         #Obstacles
-        self.display_pole = False
-        self.display_tornado = False
-        self.display_oil_spill = False
-        self.display_grease_spill = False
-        self.display_water_spill = False
+        self.enable_pole = False
```

Comment:
@shazz why did you rename the`display_pole` boolean to `enable_pole`? Same for other obstacles? It's not consistent with the cones.